diff -urN Net-SSLeay-1.88.orig/Makefile.PL Net-SSLeay-1.88/Makefile.PL
--- Net-SSLeay-1.88.orig/Makefile.PL	2019-05-18 19:21:44.521823269 +0200
+++ Net-SSLeay-1.88/Makefile.PL	2019-05-18 19:22:01.825050519 +0200
@@ -202,8 +202,8 @@
     else {
         push @{ $opts->{lib_links} },
              ($rsaref
-              ? qw( ssl crypto RSAglue rsaref z )
-              : qw( ssl crypto z )
+              ? qw( ssl crypto RSAglue rsaref )
+              : qw( ssl crypto )
              );
 
         if (($Config{cc} =~ /aCC/i) && $^O eq 'hpux') {
