--- Net-SSLeay-1.36/inc/Module/Install/PRIVATE/Net/SSLeay.pm~	2011-01-30 01:12:37.300174420 +0100
+++ Net-SSLeay-1.36/inc/Module/Install/PRIVATE/Net/SSLeay.pm	2011-01-30 01:12:42.141086087 +0100
@@ -95,8 +95,8 @@
         $opts->{optimize} = '-O2 -g';
         push @{ $opts->{lib_links} },
              ($rsaref
-              ? qw( ssl crypto RSAglue rsaref z )
-              : qw( ssl crypto z )
+              ? qw( ssl crypto RSAglue rsaref )
+              : qw( ssl crypto )
              );
 
         if (($Config{cc} =~ /aCC/i) && $^O eq 'hpux') {
